test/ejabberd_tests/tests/login_SUITE.erl

```diff
@@ -213,11 +213,11 @@ end_per_testcase(CaseName, Config) ->
 register(Config) ->
     [{Name1, UserSpec1}, {Name2, UserSpec2}] = escalus_users:get_users([alice, bob]),
     [{_, AdminSpec}] = escalus_users:get_users([admin]),
-    [Username1, Server1, _Pass1] = escalus_users:get_usp(Config, UserSpec1),
-    [Username2, Server2, _Pass2] = escalus_users:get_usp(Config, UserSpec2),
+    [Username1, _Server1, _Pass1] = escalus_users:get_usp(Config, UserSpec1),
+    [Username2, _Server2, _Pass2] = escalus_users:get_usp(Config, UserSpec2),
     [AdminU, AdminS, AdminP] = escalus_users:get_usp(Config, AdminSpec),
 
-    ok = escalus_ejabberd:rpc(ejabberd_auth, try_register, [AdminU, AdminS, AdminP]),
+    escalus_ejabberd:rpc(ejabberd_auth, try_register, [AdminU, AdminS, AdminP]),
```

Comment:
It was temporary - later commits introduced mongoose_stanza:to_map/1 which allows to check whether the return was correct, so those two lines because:
```
Res = escalus_ejabberd:rpc...
#{} = mongoose_stanza:to_map(Res)
```